app/Modules/Admin/Setting.php

```diff
@@ -551,6 +560,7 @@ public function settings_select_callback($args)
         $html .= '</select>';
         $html .= '<small class="form-text text-muted" for="smartpay_settings[' . smartpay_sanitize_key($args['id']) . ']"> ' . wp_kses_post($args['desc']) . '</small>';
 
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- The generated output has already escaped.
         echo $html;
```

Comment:
Instead of suppressing the warning, you could have just added another escape.


```suggestion
        echo wp_kses_post($html);
```

Reply:
> Instead of suppressing the warning, you could have just added another escape.

`wp_kses_post` allows some safe tags. But in some places Script tag will pass, such as allowing a modal opening. If I wrap with `wp_kses_post`, then functionality will break. I could use `wp_kses` and manually allow the tags, but it seems lot of work as I must allow the `script` tag